src/iso9660.rs

```diff
@@ -522,3 +519,48 @@ fn eat(buf: &mut Bytes, n: usize) -> &mut Bytes {
     buf.advance(n);
     buf
 }
+
+/// Parse path into a PathBuf with zero or more Normal components.  Convert
+/// path to relative and resolve all "." and ".." components.
+fn canonicalize_path(s: &str) -> Result<PathBuf> {
```

Comment:
Should we just return a `Vec<&OsStr>` instead and then we can drop the `else` branch with the `panic!()` in `get_path`?

Reply:
Good call.